packages/lit-helpers/package.json

```
@@ -30,8 +30,8 @@
     "helpers"
   ],
   "peerDependencies": {
-    "lit-element": "^1.0.0",
-    "lit-html": "^1.0.0"
+    "lit-element": ">= 1 < 3",
```

why not just?
```suggestion
    "lit-element": "^2.0.0",
```

as lit-element 1.x is something completely different (pre polymer team - not using lit-html) 🤔

---

updated